  unittest1.cpp
D:\Data-Structures\Lab 4\RecursionList\RecursionList\RecursionList.h(47,25): warning C4251:  'PallindromeByStack::stack': class 'std::stack<char,std::deque<_Ty,std::allocator<_Ty>>>' needs to have dll-interface to be used by clients of class 'PallindromeByStack'
D:\Data-Structures\Lab 4\RecursionList\RecursionList\RecursionList.h(47,25): warning C4251:         with
D:\Data-Structures\Lab 4\RecursionList\RecursionList\RecursionList.h(47,25): warning C4251:         [
D:\Data-Structures\Lab 4\RecursionList\RecursionList\RecursionList.h(47,25): warning C4251:             _Ty=char
D:\Data-Structures\Lab 4\RecursionList\RecursionList\RecursionList.h(47,25): warning C4251:         ]
D:\Data-Structures\Lab 4\RecursionList\RecursionList\RecursionList.h(47): message :  see declaration of 'std::stack<char,std::deque<_Ty,std::allocator<_Ty>>>'
D:\Data-Structures\Lab 4\RecursionList\RecursionList\RecursionList.h(47): message :         with
D:\Data-Structures\Lab 4\RecursionList\RecursionList\RecursionList.h(47): message :         [
D:\Data-Structures\Lab 4\RecursionList\RecursionList\RecursionList.h(47): message :             _Ty=char
D:\Data-Structures\Lab 4\RecursionList\RecursionList\RecursionList.h(47): message :         ]
D:\Data-Structures\Lab 4\RecursionList\RecursionList\RecursionList.h(51,1): warning C4275:  non dll-interface class 'std::exception' used as base for dll-interface class 'MyException'
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.22.27905\include\vcruntime_exception.h(44): message :  see declaration of 'std::exception'
D:\Data-Structures\Lab 4\RecursionList\RecursionList\RecursionList.h(50): message :  see declaration of 'MyException'
  UnitTest1.vcxproj -> D:\Data-Structures\Lab 4\RecursionList\x64\Debug\UnitTest1.dll
